app/javascript/packs/main_vue.js

```diff
@@ -206,6 +206,36 @@ document.addEventListener('turbolinks:load', () => {
       }
     });
   });
+
+  $('#increase-blog-font-size-btn').on('click', function() {
+    var originalFontSize = $('.wordpress-theme,h2,h3,h4,h5').not('#post-cards-section').css('font-size');
```

Comment:
Não precisa do not se usar a classe .wordpress-theme, pq tu pode aplicar somente dentro dela as mudanças. No $ tu pode selecionar só os filhos dessa classe com um $('.wordpress-theme p'), por exemplo, pra pegar todos os paragrafos... ou varias coisas com o $('.wordpress-theme p,h2,h4')